server/routes/private.js

```diff
@@ -7,6 +7,7 @@ const router = express.Router();
 const authenticateHandler = require('../models/handlers/authenticate');
 const appointmentHandler = require('../models/handlers/appointment');
 const workerTimeslotHandler = require('../models/handlers/workerTimeslot');
+const TimeslotHandler = require('../models/handlers/timeslot');
```

Comment:
Hi Peter,
To be consistent, I would recommend naming this variable to also follow lowerCamelCase as well.

Reply:
Fixed!